bin/alluxio-monitor.sh

```diff
@@ -53,13 +53,6 @@ get_env() {
   . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
   CLASSPATH=${ALLUXIO_CLIENT_CLASSPATH}
   ALLUXIO_TASK_LOG="${ALLUXIO_LOGS_DIR}/task.log"
-
-  # Remove the remote debug configuration to avoid the error: "transport error 20: bind failed: Address already in use." 
-  # See https://github.com/Alluxio/alluxio/issues/10958
-  ALLUXIO_MASTER_MONITOR_JAVA_OPTS=$(echo ${ALLUXIO_MASTER_JAVA_OPTS} | sed 's/^-agentlib:jdwp=transport=dt_socket.*address=[0-9]*//')
```

Comment:
uhh but these are being used later in this file? how is this scenario properly handled now?

Reply:
The idea is that `-agentlib` args have been moved to ALLUXIO_MASTER_ATTACH_OPS to avoid this issue. Moreover, the regex is incorrect and does not appropriately remove what it needs to.

Reply:
i'm referring to how to handle these lines that call into these. with the code as it currently is, this probably doesn't work when `ALLUXIO_MASTER_MONITOR_JAVA_OPTS` is not set
```
master)
      monitor_exec=alluxio.master.AlluxioMasterMonitor
      alluxio_config="${alluxio_config} ${ALLUXIO_MASTER_MONITOR_JAVA_OPTS}"
      ;;
```

Reply:
I restore the `MONITOR_OPTS` as they were before. They still need the `sed` command on the off chance that `-agentlib` slips into `JAVA_OPTS` somehow.